apps/system-e2e/src/tests/islandis/service-portal/acceptance/mocks/licenses.mock.ts

```diff
@@ -0,0 +1,581 @@
GQTE9zzzx9K8SLKzOVYbT09xXn42P73m7nr4OX7vl7HD6hpv2GdlwTE33SOcexrPltypBGA4GARXZ6lCGG2Q7oz7dK5m7gMDnK9eh7Vwcp2GNMrq5OPm/Q00XskbfMSD9KuupJPHNRMinOevtVp2IYyPV2HRtufanjVyuBu61EYI05wKia3jIZRleeMDODWtyHcuDWMKRuyDSnWCwwCP61RNsrAhs7hxuFILY7uPmXpn0FVzEasvJfMSct15pVvm3D5uagW2GeX4/SrcNukfIHH6mjmBRuPjvHAVcHnvUsa7/mcn1wRSCRBjjdg9qmhTeu7LE9hjilzdx2sSxgsMNyPYVIkXm/MzkBf4R3pYwG+Ubi3XrxmrUVszDjnHVjUMqKFtEJAAAHtir8Nt5Jy2AvrmnW0PIJGQR3PSrsUKgjOW74pGqQlvbncGYZQ9F9KuwhI2+VSx60kS7hhT074q1BAAGO75j0ApGqjoPjt/OYF8g+g6VdklwgVBt28cCqzqwjjCkoc8vihejLk4Oc45zSdjdEszDCsX3Y7Y6VXYsW243gdQOmTSllICn7x7HtT4/9HwqnLOalq6L2LEcflr+8JfPOB2rRso1bD8gdBntWah2o25zknJGav2TrJzjkDIycVkT0uaUZI43HGee+TWnZFCPMc7VQZJ7msy2gkLA5DrjFUvFV3Nb6TDYWokku9QlEEUcRw5J67foMmr5dbGd+5sfB7R7fWdZ1bW9RnOngsVtr2NuExwqkHqM9R/KuuudUkvsW19ciaVf9XOBt6djUml+H4vh74Gj064IuJrmMNBM0GHjUgZDerdeRWHKd11BCWLg4QEDOOe1e/Th7Omkz52rL2tRtG7a3T4m+0hUVSoTPBZcjJH0yD+FfOv7W3jZ5bu38MLc+eLXdOxAx94/IPwX+de++J9ag8EeEbvUprplWwBkkUfMJQV2qikj5cngj2r4A8b+Jrrxb4kvdVu3DS3Mm4heAB2A+g4p7RuZt9EY8R3y5JxWnbnZySAfQ1loOfk61q2qllUk4Pc1BBqQY2jA/wNTggdBiqUcoU4B75HoKmeUdj1rQTLRcHqd3r7Unmc5B5HTP5VUDqo5PPrUbXOcnOTQSblteKw2scfh+oqeTLpvHzZGQR0xWBb3eD179fSrw1PaMHgdAD3oeqA/SkkgcjjvUfc471YZgUyASOxqAplsntXwh9YRNwMkkY64qjPbm4lBH3RViWQySYOQKlVRtzSvbYohEYjQcYNQu4B5HNTzPgZ6ZqDyxM3IP4VIAEEy56cU15Vgbjrip/L2L1JqGaEytjB+tMBUuGm5J49KnicISetQKgQYAIqQRjaOWqgLyzl+pqpr3ifT/AAzp8t7qE6QQIuSXPX2FcL8UvjFonwnsR/aUry37pujsoVJdvQnsB718XfFP46a78SNQZrh/s1kCTFbRn5VH9T7134fCSre89jir4iNP3U9T2H4u/tNyas0llo+6G3PymQnDNXzxrPil7qVmkfc785Nc61253OxZm96pyZdsnJP0r6CnRhSVoo8edWVR3ZpvqbTNhTj3HrWdPdGZyoYn39aYo2oxA2npV3SrGNszSk+UnU46+1bkRK8Nmz/MSQvqaU7UBC84/iqze3CyDEQIT2qkzfLtB/Cnclu4I53fyq0JGxnJ57VWGFVSev0qSSTC9f0oQN3G3MoKkZ5NVOOxp7/MxqM4PekIEXc/tmv0z/Zn0A+DvgPpRnTyZrpGnbPUhicfpivz3+F/hF/HPjzSNFiVmFxOqysP4Uz8x/Kv0/8AGkCeG9A0/TrN1FvGiQoq+gGK6aUdHIzk7Ox5v4julkE0MhBVwQQeu09a8cjg+yTTW5OWgcxnP6H8q9L8VXStcqhOGH864XxBYSw3VtesuIrzdGc8ZdOufqpH5V5uJV1c9LCytKxRu7YSQcYP9K5y/tfmKZOzOa6hJFCFHYcevFUL6FXUjAyfevObsetY4u4sypJUYHWqhtyeMbc+3JroLy3ZMkHj07VTa23L13E9TS5ibGK8RHGfw7mgqSNwGPUVqSW2eVx9cVXaAocN36VonczsUjEFBBYjPr2qPaUHPYfw1oGEkYOAPrzUPklSRxTHYgiV2OTnI6A1Ku5mwCc+1SCI4BUBVFWo4iEBQYyO45xSuFiGCDBC8g+g5q2kZl2gfJjsO/uanhs3RcgAk9z1rUtdMKKCT8x6g0AQW9ioAaRRn1BrTgt1UDK7sDj0qWK22Mc9B6ipVVnGBwPamVYhjJYgAZPt1q2kLNgk8jn60+3tQrZbueT/AEq0EKHGAQOmBQaRXQSIELz16Zqe3j2StjGw9yOlOgiDn5sY9DU0twsShT857KKk1B4wWyzHAPAPQVXlugvyocnOAwHSmPKbjO75VB7cZpUHORjAH1rNvoaBbQLGGJY785JPcGpi53ZUAsOMgdKh2lcMSCQc8VOpwgI4GfxqLjJYLcAZXLZyTuPSr1pERjBBJI4A6VBbKcAg5Pc5rRt2WMYBwe4x3pg3Y0bUMroSVHUlsZ4rndLWbxl8R44baH7VHpMD3ByxVQ/QEkdAKm8T62PD+hzzMdp24HXkntUvwp0UWnhF9fk3Q399K7K5YqwQdjzyOOh9a7sNBVKl3sjy8XVVOLS3Z2+seLLi+tEsZi4e3RVRJTlgcZOD3U84qno6M4aRlJHy7uTlRn7w9ceneuRl1R77WE88Foicbl6jnqK6+/8AEEXhDQb3WpcRpbxl2QnPmnHy7fc16j1lboePFpanjn7VPjx7LTrLwnBcozbRPOiDG1Qcxr/M18t7yzEng+tbHjrxRd+LfEl5ql6d89zJvYjt2A/AViq/yk1MndkdLksX+sBHFaUMhC9azYSxxwfyq4CQBjpigC6JMv1qRpAmcdap+ZtAz1pJpCpUnvViZaklOPl4FRGYDODn2qu0pIxnP4VAZGkfPapYI0IJSevy+uafLOAVxjHtVKKfLbF6GmvMWVQBnJwMDjPpRLYR/9k=',
```

Comment:
Do we need this? Can we make it a lot smaller?

Reply:
probably unnecessary, i'll remove